amplipi/updater/asgi.py

```diff
@@ -329,7 +329,8 @@ def request_support():
       '/opt/support_tunnel/venv/bin/python3 -m invoke request'.split(),
       capture_output=True,
       cwd='/opt/support_tunnel',
-      timeout=120
+      timeout=120,
+      check=True
```

Comment:
putting `check=True` here handles this case just a smidge better; it's not strictly necessary.